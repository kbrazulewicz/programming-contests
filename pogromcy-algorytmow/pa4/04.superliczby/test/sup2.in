50
31 7 28 18 50 14 3 8 11 33 6 32 42 47 37 9 19 27 1 23 15 39 38 10 25 34 20 16 30 46 35 48 36 26 44 4 13 45 40 21 22 5 12 24 29 2 49 43 17 41
