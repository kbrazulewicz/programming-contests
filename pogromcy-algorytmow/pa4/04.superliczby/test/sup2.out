16
3 6 7 8 9 11 19 23 25 30 34 35 36 44 45 49
